gamemode/modules/commands/sv_commands.lua

```diff
@@ -62,6 +62,26 @@ commands.addType( "string", function( arg )
     return arg
 end )
 
+commands.addType( "entity", function( arg, ply ) 
+    local n, err = commands.types["int"]( arg, ply )
+    if err == nil then
+        return Entity( n ), nil
+    end
+
```

Comment:
Could we get a option to get self here?
```suggestion
    if arg == "^" then
        return ply
    end
```
Something like that